rootfs/etc/nginx/lua/test/certificate_test.lua

```diff
@@ -0,0 +1,61 @@
+_G._TEST = true
+
+local certificate = require("certificate")
+local unmocked_ngx = _G.ngx
+
+describe("Certificate", function()
+  describe("call", function()
+    local ssl = require("ngx.ssl")
+    local match = require("luassert.match")
+
+    ssl.server_name = function() return "hostname" end
+    ssl.clear_certs = function() return true, "" end
+    ssl.cert_pem_to_der = function(cert) return cert end
```

Comment:
please make sure you are mocking these function by following their original API properly: https://github.com/openresty/lua-resty-core/blob/master/lib/ngx/ssl.md#cert_pem_to_der
for example this function specifically, return two values not just one